Lib/_pyio.py

```diff
@@ -242,7 +244,7 @@ def open(file, mode="r", buffering=-1, encoding=None, errors=None,
             buffering = -1
             line_buffering = True
         if buffering < 0:
-            buffering = raw._blksize
```

Comment:
I'd make it named `_MAXIMUM_BUFFER_SIZE` (underscore leading should make it private), then removing the export from `_io` I think will pass tests.

I think the checks are 1. `_all_` contains everything without an underscore, 2. `_all_` matches between `_io` and `_pyio`

https://github.com/python/cpython/blob/main/Lib/test/test_io.py#L4216-L4219 which uses https://github.com/python/cpython/blob/main/Lib/test/support/__init__.py#L1627-L1683